lib/cyclonedx/schema/bom-1.3.schema.json

```diff
@@ -0,0 +1,1054 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
The [schema](https://github.com/CycloneDX/specification/blob/master/schema/bom-1.3.schema.json) provided uses draft-07 compared to draft-04 which is the maximum version our `json-schema` gem supports. Potentially in another PR we'd like to support draft-04 and higher with this [gem](https://github.com/davishmcclurg/json_schemer).